src/main/java/com/minecolonies/coremod/network/messages/ColonyStylesMessage.java

```diff
@@ -55,11 +76,39 @@ public void fromBytes(@NotNull final ByteBuf buf)
         return map;
     }
 
+    @NotNull
+    private static Map<String, String> readMD5MapFromByteBuf(@NotNull final ByteBuf buf)
```

Comment:
Again converting this to a final BiMap would improve performance and readability.